packages/gmmlib.rb

```diff
@@ -0,0 +1,32 @@
+require 'package'
+
+class Gmmlib < Package
+  description 'The Intel(R) Graphics Memory Management Library provides device specific and buffer management for the Intel(R) Graphics Compute Runtime for OpenCL(TM) and the Intel(R) Media Driver for VAAPI.'
+  homepage 'https://github.com/intel/gmmlib/'
+  compatibility 'i686, x86_64'
+  @_ver = '20.4.1'
+  version @_ver
+  source_url "https://github.com/intel/gmmlib/archive/intel-gmmlib-#{@_ver}.tar.gz"
+  source_sha256 'd54d547f9f9e74196dead6a338923039ea10c859f1f693f33f10be1562b81d6d'
+  
+  depends_on 'libva'
+  
+  def self.prebuild
```

Comment:
Since this is using sed to replace text, a better section would be `def self.patch`.